### ISSUE TITLE
thermal printers

### ISSUE THREAD

**Issue by EmanueleFeola** [tojocky/node-printer#299]
Does this library work to print text with thermal printers?
Has anyone tried yet?

**Comment by rkcth** [comment]
I use this with both Epson thermal receipt printers and zebra direct thermal label printers.  Unfortunately, it doesn't appear to be being kept up to date and its kind of hard to find the exact version of electron to get it working.

**Comment by ssshake** [comment]
I'm trying to get this to work with my L+H2 thermal printer but for each page I print it just feed 5-6 labels through blank, never prints anything.

I would really like to get this working because I've written a tool to grab all the order from my online stores, auto-buy shipping labels and now I just need it to auto print labels and my quality of life would be so much improved!

**Comment by juliyamartoons** [comment]
Yes i Have Tried Thermal Printers[.](https://astanehconstruction.com/)
